src/System.IO.FileSystem.DriveInfo/tests/DriveInfo.Windows.Tests.cs

```diff
@@ -154,22 +154,35 @@ public void TestInvalidDiskProperties()
         [PlatformSpecific(TestPlatforms.Windows)]
         public void GetVolumeLabel_Returns_CorrectLabel()
         {
-            int serialNumber, maxFileNameLen, fileSystemFlags;
-            int volNameLen = 50;
-            int fileNameLen = 50;
-            StringBuilder volumeName = new StringBuilder(volNameLen);
-            StringBuilder fileSystemName = new StringBuilder(fileNameLen);
+            void doDriveCheck()
```

Comment:
Nit: do => Do
(though to be fair it is a "local" function and we say locals should be camel cased, but I think local functions should be named more like members than like locals)